tests/datasource/fluent/test_batch_request.py

```diff
@@ -188,21 +188,84 @@ def test_batch_request_config_serialization_round_trips(
         ),
     ],
 )
-def test_batch_request_config_partitioner_round_trip_serialization(
+def test_batch_request_config_partitioner_dict_round_trip_serialization(
     partitioner: PartitionerT,
     TypedBatchRequest: type[BatchRequest],
 ) -> None:
-    datasource_name: Final[str] = "my_datasource"
-    data_asset_name: Final[str] = "my_data_asset"
-
     batch_request = TypedBatchRequest(
-        datasource_name=datasource_name, data_asset_name=data_asset_name, partitioner=partitioner
+        datasource_name="test-datasource", data_asset_name="test-asset", partitioner=partitioner
     )
 
-    # dict
     batch_request_dict = batch_request.dict()
     assert TypedBatchRequest(**batch_request_dict) == batch_request
 
-    # json
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "partitioner,TypedBatchRequest",
+    [
+        pytest.param(
```

Comment:
I'm going to seem like a terrible person here, but we should DRY this up probably by making a helper function, assuming this list of `pytest.param` is identical to above.